contracts/cw1-subkeys/helpers.ts

```diff
@@ -151,9 +159,9 @@ interface InitMsg {
 }
 
 // TODO: define more of these
-type CosmosMsg = SendMsg; 
+interface CosmosMsg {}
```

Comment:
I would rather make this tighter check;

`type CosmosMsg = SendMsg | DelegateMsg | UndelegateMsg | RedelegateMsg | WithdrawMsg`